dot/rpc/subscription/websocket.go

```diff
@@ -353,8 +347,7 @@ func (c *WSConn) initRuntimeVersionListener(reqID float64, _ interface{}) (Liste
 
 	c.mu.Lock()
 
-	c.qtyListeners++
-	rvl.subID = c.qtyListeners
+	rvl.subID = atomic.AddUint32(&c.qtyListeners, 1)
```

Comment:
if this is atomic now, it can be outside the lock?

Reply:
I think we need the lock to guarantee the correct write into the `Subscription` map